config/packages/dev/framework.yaml

```diff
@@ -0,0 +1,2 @@
+framework:
+    ide: 'phpstorm://open?file=\\wsl$\%env(string:WSL_DISTRO_NAME)%%%f&line=%%l'
```

Comment:
Please remove this, as it seems to be specific to Windows/WSL environments.

Reply:
done